app/uk/gov/hmrc/tradergoodsprofilesdatastore/models/CheckRecords.scala

```diff
@@ -18,21 +18,29 @@ package uk.gov.hmrc.tradergoodsprofilesdatastore.models
 
 import play.api.libs.json.{Format, JsSuccess, JsValue, Json, Reads, Writes}
 
+import java.time.Instant
+
 case class CheckRecords(
```

Comment:
Given that we're repurposing this, maybe it'd better to rename it too, what about `RecordSummary`?